src/resolve-info.ts

```diff
@@ -182,18 +195,30 @@ const memoizedExpandFieldNode = memoize4(expandFieldNode);
 function expandFieldNode(
   schema: GraphQLSchema,
   fragments: FragmentsType,
-  node: FieldNode,
+  node: JitFieldNode,
   fieldType: GraphQLOutputType
 ): FieldExpansion | LeafField {
+  const shouldInclude = (__context: ShouldIncludeContext): boolean => {
```

Comment:
We should not expect the caller to pass the context. Maybe only accepting variables is good.

Reply:
I've changes the type and renamed arg to `variables`. Should be more convenient